### DISCUSSION TITLE
Delay up to 3 minutes when updating from Content Sanity to Vercel

### DISCUSSION THREAD

**Discussion by faizfx** [vercel/community#1650] [Help] [answered]
### Summary

Hi, can someone help me every time I make changes on GitHub or update anything on Sanity? It usually takes a 3-minute delay to deploy.

<img width="639" alt="image" src="https://user-images.githubusercontent.com/116898380/221916791-a0ab6942-714b-45d8-ac8e-8162cfddf12a.png">

**Comment by amyegan** [comment]
Hi @faizfx! Are you asking about the build time, or a delay when starting the build? There are many strategies for [reducing build time](https://vercel.com/guides/how-do-i-reduce-my-build-time-with-next-js-on-vercel), but deciding on the ideal solution will depend on your app and preferred workflow

**Comment by faizfx** [comment]
Im talking about builtime,
the delay happend everytime I update content on sanity the vercel will
deploy up to 3minute before I can saw the content i update on my blog.

**Comment by amyegan** [comment, marked as answer]
One common reason for slow builds is generating a lot of pages with each build. You should take a look at this guide for some more build speed improvement options: [vercel.com/guides/how-do-i-reduce-my-build-time-with-next-js-on-vercel](https://vercel.com/guides/how-do-i-reduce-my-build-time-with-next-js-on-vercel)

Any dependencies that are no longer needed should be removed, since a long dependency list can make the install step take a long time. It's also a good idea to make sure dependencies are up-to-date and sub-dependencies are not in conflict.
